gui-wm/hyprland/hyprland-9999.ebuild

```diff
@@ -77,6 +77,7 @@ DEPEND="
 BDEPEND="
 	${WLROOTS_BDEPEND}
 	|| ( >=sys-devel/gcc-13:* >=sys-devel/clang-16:* )
+	dev-cpp/tomlplusplus
```

Comment:
tomlplusplus includes headers during the build. I tested this locally and it appears to work with it only in BDEPEND

Reply:
The difference between BDEPEND and DEPEND is that BDEPEND is for packages that are applicable for CBUILD (so simply they are program that get executed) while DEPEND is for packages that are applicable for CHOST (so they are built against, like headers and libraries).

Also using meson tomlplusplus you can build a library to link against, in which case it would also belong in RDEPEND.